package.json

```diff
@@ -213,6 +218,7 @@
     "preact-compat": "^3.18.5",
     "prettier": "^2.0.1",
     "pretty-quick": "^2.0.0",
+    "puppeteer": "^2.1.1",
```

Comment:
How about `puppeteer-core`, or it would download the whole Chromium every install?